lib/v8.js

```diff
@@ -397,6 +398,24 @@ function deserialize(buffer) {
   return der.readValue();
 }
 
+/**
+ * @param {{
+ *   duration: number,
+ * }} [options]
```

Comment:
```suggestion
 * @param {{
 *   duration: number,
 * }} options
```
The `options` parameter isn't optional here because we're validating if it's value is an object.